.ruby-version

```diff
@@ -1 +1 @@
-2.2.4
\ No newline at end of file
+2.7.5
```

Comment:
@jeremygottfried is there a reason we're keeping this at Ruby 2.x? If so, we should probably use Ruby 2.7.7, which is the latest release.